includes/classes/db/mysql/query_factory.php

```diff
@@ -172,7 +172,13 @@ function show_error() {
     echo '</div>';
   }
 
-  function Execute($zf_sql, $zf_limit = false, $zf_cache = false, $zf_cachetime=0, $remove_from_queryCache = false) {
+  function Execute($zf_sql, $zf_limit = false, $zf_cache = true, $zf_cachetime=3600, $remove_from_queryCache = false) {
+     if (strtoupper(substr($zf_sql,0,6)) != 'SELECT') {
+         $zf_cache = false; 
+     }
+     if (defined('IS_ADMIN_FLAG') && IS_ADMIN_FLAG == 'true') {
+         $zf_cache = false; 
+     }
```

Comment:
Given this change, how would an admin-side query be executed WITH cache when desired?

Reply:
My suggestion would be to cross that bridge in phase 2.  Perhaps another parameter saying force or something?  I'd like to be sure we get the catalog side implementation right before worrying about admin.  Can you think of an admin case where caching would be desirable, given that there could be two admins working in the system?